operator-wg/whitepaper/051_kubebuilder.md

```diff
@@ -0,0 +1,17 @@
+## kubebuilder Operator Framework
+
+The kubebuilder framework provides developers the possibilities to extend the Kubernetes API by using Custom Resource Definitions, and to create operators that handle these custom resources.
+
+The main concept provided by the kubebuilder framework is a *Manager*. In the same way the native Kubernetes controllers are grouped into a single Kubernetes Controller Manager (`kube-controller-manager`), you will be able to create several operators and make them managed by a single manager.
```

Comment:
Disagree that this is the "main concept". It's certainly one of them but the main twp concepts are "apis" and "controllers".

Reply:
Have to agree with @coderanger.

Reply:
Main entry point?

Reply:
It is the main entry point but I don't know how relevant is this fact for a short description of the project. Maybe if you change it to this so that the "controller" concept is also present:
```suggestion
The main entry point provided by the kubebuilder framework is a *Manager*. In the same way the native Kubernetes controllers are grouped into a single Kubernetes Controller Manager (`kube-controller-manager`), you will be able to create several contrllers and make them managed by a single manager.
```